fastparquet/api.py

```diff
@@ -203,6 +204,34 @@ def filter_out_stats(rg, filters, helper):
     return False
 
 
+def statistics(obj):
+    if isinstance(obj, parquet_thrift.ColumnChunk):
+        md = obj.meta_data
+        s = obj.meta_data.statistics
```

Comment:
statistics may be None, but otherwise no. All four fields within statistics are also optional, and so may be None.